dali/test/python/test_pipeline.py

```diff
@@ -2333,3 +2332,61 @@ def def_ref():
         (ref,) = ref_pipe.run()
         check_batch(cpu, ref, bs, 0, 0, "HWC")
         check_batch(gpu, ref, bs, 0, 0, "HWC")
+
+
+def test_cse():
```

Comment:
How about adding a test with External Source (to make sure it's not eliminated) and manually named operators?

Reply:
Manually named operator: see line 2343.
ES - we already have tests with multiple external sources and they did fail before adding the fix, so I'd call it covered.